site-src/guides/getting-started-latest.md

```diff
@@ -193,6 +193,72 @@ kubectl apply -k https://github.com/kubernetes-sigs/gateway-api-inference-extens
          kubectl get httproute llm-route -o yaml
          ```
 
+=== "NGINX Gateway Fabric"
+
+      NGINX Gateway Fabric is an implementation of the Gateway API that supports the Inference Extension. Follow these steps to deploy an Inference Gateway using NGINX Gateway Fabric.
+
+      1. Requirements
+
+         - Gateway API [CRDs](https://gateway-api.sigs.k8s.io/guides/#installing-gateway-api) installed (Standard or Experimental channel).
+         - [Helm](https://helm.sh/docs/intro/install/) installed.
+         - A Kubernetes cluster with LoadBalancer or NodePort access.
+
+      2. Install the Inference Extension CRDs
+         
+         ```bash
+         kubectl kustomize "https://github.com/nginx/nginx-gateway-fabric/config/crd/inference-extension/?ref=v2.2.0" | kubectl apply -f -
```

Comment:
The guide is already installing the inference CRDs. Additionally, this version (latest/main) of the guide installs the inference CRDs from the `main` branch and not v1.0.2 as referenced by `https://github.com/nginx/nginx-gateway-fabric/blob/v2.2.0/config/crd/inference-extension/kustomization.yaml`